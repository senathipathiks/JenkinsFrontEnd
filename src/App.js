//import logo from './logo.svg';
import './App.css';

function App() {
  return (
    <div className="App">
      <h1>Welcome to My Page !!!</h1>
      <br />
     <h2> Hi Welcome to Rele</h2>
    </div>
  );
}

export default App;
