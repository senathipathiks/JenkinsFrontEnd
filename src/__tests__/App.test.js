import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer'
import App from '../App';

describe('Test Suite for App',()=> {

  
  test("Snapshot test ", () => {
    //render(<App />);
    const comp = renderer.create(<App />)
    const tree=comp.toJSON();
    expect(tree).toMatchSnapshot();
  });

  // it('renders welcome to Relevantz', () => {
  //   render(<App />);
  //   const linkElement = screen.getByText("Hi Welcome to Relevantz");
  //   expect(linkElement).toBeInTheDocument();
  // });
})

