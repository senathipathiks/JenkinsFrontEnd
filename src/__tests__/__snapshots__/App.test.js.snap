// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Suite for App Snapshot test  1`] = `
<div
  className="App"
>
  <h1>
    Welcome to My Page !!!
  </h1>
  <br />
  <h2>
     Hi Welcome to Rele
  </h2>
</div>
`;
